src/commissaire_service/clusterexec/__init__.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, exchange_name, connection_url):
             {'routing_key': 'jobs.clusterexec.*'}
         ]
         super().__init__(exchange_name, connection_url, queue_kwargs)
+        self.storage = StorageClient(self)
```

Comment:
There's a few different ways I could have integrated the client API.  All would work.  I guess the only reason I didn't add it to `CommissaireService` directly is the `StorageService` is also a `CommissaireService` and the client API wouldn't make sense there.  It's a weak reason though; if you'd prefer it done differently I can change it.